examples/mysql/all_about_inserts/src/lib.rs

```diff
@@ -1,9 +1,5 @@
-extern crate chrono;
 #[macro_use]
 extern crate diesel;
```

Comment:
As diesel (master) now supports explicitly importing derive/… macros I think we should also fix that as part of this PR. 

(Downside is that there will probably even more confusion about what's supported in the last release and what not)

Reply:
Good point, removed via 4e0b9a3. There're still some `extern crate`s since it seems `print_scheme` isn't _migrated_ to edition 2018.